src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const authEmulatorHost = 'http://localhost:9099/'
+export const functionsEmulatorHost = 'http://localhost:5050'
```

Comment:
retrieve from `.env` config file ?

Reply:
is it really needed ? The info won't change by environment, since it is only used locally when the `REACT_APP_AUTH_EMULATOR_ENABLED` is `true`. Furthermore, unless the developer changes the default values, firebase emulators should always use the same ones